cms/templates/partials/admissions-section.html

```diff
@@ -0,0 +1,61 @@
+{% load image_version_url wagtailcore_tags humanize %}
+<div class="container-fluid admissions-section">
+    <div class="row">
+        <img class="admissions-image" src="{% image_version_url page.admissions_image 'fill-1920x540' %}" alt="Admissions" />
+    </div>
+    <div class="row mb-4">
+        <div class="col-md-10 offset-md-2 col-sm-12 offset-sm-0 pt-5 bg-white">
+            <div class="container-fluid admissions-section">
+                <div class="row">
+                    <div class="col-md-4 col-sm-12 mb-2">
+                        <h1 class="section-heading">How to Apply</h1>
+                        <h2 class="admissions-heading mt-2"><a href="/admissions">Admissions</a></h2>
```

Comment:
Do we know that this is supposed to be a link? Is the `/admissions` page something that we're going to implement (or have already implemented)?

Reply:
We're going to implement this, I've also requested clarification on this on the invision design from Peter.